README.md

```diff
@@ -140,6 +140,64 @@ by user is not implemented [#324].
 
 [#324]: https://github.com/Taxel/PlexTraktSync/issues/324
 
+## Docker Compose
+
+This will show you how to set up PlexTraktSync for one user on a Plex Media Server for Docker Compose.  Duplicate this for each additional user wanting to sync to Trakt.
+
+### Setup
+
+First we need to create the configuration files needed.  This can be done by running the following command line...
+```bash
+docker run \
+  -it --rm \
+  --name plextraktsync-john \
+  --network t2_proxy \
+  -v /home/user/docker/appdata/plextraktsync-john/config:/app/config \
+  -e PTS_CONFIG_DIR=/app/config \
+  -e PTS_CACHE_DIR=/app/config \
+  -e PTS_LOG_DIR=/app/config \
+  ghcr.io/taxel/plextraktsync:latest
+```
+|Command|Explanation  |
+|:--:|--|
+| ```-it``` | Runs session as interactive |
+| ```--rm``` | Removes the container once the session is closed |
+| ```--name plextraktsync-john``` | Naming the session helps diagnosing issues if they happen |
+| ```--network t2_proxy``` | Name of the same docker network Plex Media Server is on, remove if nessessary |
+| ```-v /home/user/docker/appdata/plextraktsync-john/config:/app/config``` | Before ":" is the path to a unique folder for this Plex user |
+| ```-e PTS_CONFIG_DIR=/app/config```<br>```-e PTS_CACHE_DIR=/app/config```<br>```-e PTS_LOG_DIR=/app/config``` | Environment variables as per ```PlexTraktSync/docker-compose.yml``` example |
+
+Keep running this until you've completed your first successful sync between Plex Media Server and Trakt.  That should be three times...
+
+ 1. Setup link with Plex Media Server
+ 2. Setup link with Trakt
+ 3. Sync between Plex Media Server and Trakt.
+
+### Docker Compose
+
+Once you have your configuration files created, you need to make sure they have the correct permissions to be used by docker compose as they've probably been created with ```root:root```.
+
+Docker normally uses the local user and the docker group, so run ```sudo chown user:docker /home/user/docker/appdata/plextraktsync-john -R``` .  Check your own setup incase it is different.
+
+You can now set up a docker container in your docker compose file that will scrobble that Plex user's activities to
+```yaml
+  plextraktsync-john:
+    image: ghcr.io/taxel/plextraktsync:latest
+    container_name: plextraktsync-john
+    networks:
+      - t2_proxy
+    depends_on:
+      - plex
+    restart: unless-stopped
+    volumes:
+      - $DOCKERDIR/appdata/plextraktsync-john/config:/app/config
+    environment:
+      - PTS_CONFIG_DIR=/app/config
+      - PTS_CACHE_DIR=/app/config
+      - PTS_LOG_DIR=/app/config
+    entrypoint: ["python3", "-m", "plex_trakt_sync", "watch"]
```

Comment:
I suggest to show a `sync` command as example instead of `watch`.
`sync` is the standard command for full plex->trakt and trakt->plex status update.
`watch` is a specific command.

Reply:
Isn't ```sync``` a once off, ie will close the container after it completes once?  And ```watch``` keeps watching and reporting progress and completion, keeping the container live?

Reply:
Yes, but `watch` does not perform a full sync.
I think the example given in readme should be standard as much as possible.
I suggest to remove `watch` from entrypoint (just like [Dockerfile](https://github.com/Taxel/PlexTraktSync/blob/main/Dockerfile) is, no command means sync).
[Readme](https://github.com/Taxel/PlexTraktSync#setup) explains the use of a cronjob to repeat the sync.

**Keeping container alive :** @Taxel [talked](https://github.com/Taxel/PlexTraktSync/discussions/88) about a cronjob inside the container to keep it alive and make it run `sync` every x hours. It's a good idea, maybe for a next PR.

Reply:
A full sync is performed on the third run of the container in the Setup phase.
I then run the container in watch mode from Docker Compose as that suits my neads.
Feel free to modify what I've written.